RiotSharp/MatchEndpoint/MatchReference.cs

```diff
@@ -23,14 +23,14 @@ public class MatchReference
         /// <summary>
         /// The match ID relating to the match.
         /// </summary>
-        [JsonProperty("matchId")]
-        public long MatchID { get; set; }
+        [JsonProperty("gameId")]
+        public long GameId { get; set; }
 
         /// <summary>
         /// The ID of the platform on which the game is being played
         /// </summary>
         [JsonProperty("platformId")]
-        public string PlatformID { get; set; }
```

Comment:
isn't the converter needed?

Reply:
This could explain the bogue in the Example program.

Reply:
I have no error about this so I don't think so 🤷‍♂️

Reply:
mmmh

Reply:
I added it in the dbae593 commit.

Reply:
I don't know why it's not updated here 🤷‍♂️

Reply:
yeah the review is a bit finicky when you change something just below where the commit is